homeassistant/components/zha/core/device.py

```diff
@@ -233,32 +222,25 @@ def get_cluster_listener(self, name):
 
     async def get_clusters(self):
         """Get all clusters for this device."""
-        clusters = {}
-        for ep_id, endpoint in self._zigpy_device.endpoints.items():
-            if ep_id == 0:
-                continue
-            clusters[ep_id] = {
+        return {
+            ep_id: {
                 IN: endpoint.in_clusters,
                 OUT: endpoint.out_clusters
-            }
-        return clusters
+            } for (ep_id, endpoint) in self._zigpy_device.endpoints.items()
```

Comment:
Now we're not checking for `ep_id` 0?

Reply:
good catch. Fixed. Doesn't break anything, there is just no point in returning ep 0 because it is the ZDO (zigbee device object) endpoint and it doesn't have clusters so there is no point in including it.